CHANGELOG.md

```diff
@@ -3,6 +3,46 @@
 All notable changes to this project will be documented in this file.
 This project adheres to Semantic Versioning.
 
+## [1.7.0] 2021-07-15
+
+### Added
```

Comment:
- Add filtering and grouping to goroutines command  (@aarzilli)

Reply:
How did I miss that?! Thank you!